src/Moryx.Products.Management/Implementation/RecipeStorage.cs

```diff
@@ -198,7 +198,7 @@ public static WorkplanEntity ToWorkplanEntity(IUnitOfWork uow, Workplan workplan
                 var reference = referenceRepo.Create((int)WorkplanReferenceType.NewVersion);
                 reference.Source = workplanEntity;
                 reference.Target = workplanEntity = workplanRepo.Create(workplan.Name, workplanEntity.Version + 1, (int)workplan.State);
-                EntityIdListener.Listen(workplanEntity, workplan);
+                
```

Comment:
Don't we need to link them here now? (Also for all other changes)

Reply:
Since the EntityIdListener didn't work and the programm itself works, it should be fine

Reply:
Since the EntityIdListener didn't work and the programm itself works, it should be fine